wallet/reference/multichain-api.md

```diff
@@ -0,0 +1,396 @@
+      "expiry": "2022-12-24T17:07:31+00:00",
+      "caip154-mandatory": "true"
+    }
+  }
+}
+```
+
+</TabItem>
+<TabItem value="Result">
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "sessionScopes": {
+      "eip155": {
+        "references": ["1", "137"],
+        "methods": ["eth_sendTransaction", "eth_signTransaction", "get_balance", "eth_sign", "personal_sign"],
+        "notifications": ["accountsChanged", "chainChanged"],
+        "accounts": ["eip155:1:0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb", "eip155:137:0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb"]
+      },
+      "eip155:10": {
+        "methods": ["get_balance"],
+        "notifications": ["accountsChanged", "chainChanged"],
+        "accounts": []
+      },
+      "eip155:42161": {
+        "methods": ["personal_sign"],
+        "notifications": ["accountsChanged", "chainChanged"],
+        "accounts": ["eip155:42161:0x0910e12C68d02B561a34569E1367c9AAb42bd810"]
+      },
+      "eip155:0": {
+        "methods": ["wallet_getPermissions", "wallet_creds_store", "wallet_creds_verify", "wallet_creds_issue", "wallet_creds_present"],
+        "notifications": []
+      }
+    },
+    "sessionProperties": {
+      "expiry": "2022-11-31T17:07:31+00:00",
+      "globalConfig": {
+        "foo": "bar"
+      }
+    }
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+### `wallet_getSession`
+
+Gets the scopes and properties within the active connection.
+This method is defined in [CAIP-312](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-312.md).
+
+:::note
+MetaMask doesn't support session IDs.
+:::
+
+#### Parameters
+
+None.
+
+#### Returns
+
+The scopes and properties of the connection.
+
+#### Example
+
+<Tabs>
+<TabItem value="Request">
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "wallet_getSession",
+  "params": {}
+}
+```
+
+</TabItem>
+<TabItem value="Result">
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "sessionScopes": {
+      "eip155:1": {
+        "methods": ["eth_signTransaction"],
+        "notifications": ["accountsChanged"],
+        "accounts": ["eip155:1:0xabc123"]
+      },
+      "eip155:137": {
+        "methods": ["eth_sendTransaction"],
+        "notifications": ["chainChanged"],
+        "accounts": ["eip155:137:0xdef456"]
+      },
+      "solana:4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ": {
+        "methods": ["getBalance", "getAccountInfo", "sendTransaction", "getBlock"],
+        "notifications": [],
+        "accounts": ["solana:4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ:4Nd1mS8AUwK3kU3gdiAM6QCvqhA7Do8rKtMXsGyqrJxy"]
+      }
+    }
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+### `wallet_invokeMethod`
+
+Invokes the specified [JSON-RPC API](json-rpc-methods/index.md) method on the specified network
+previously authorized to the dapp within a connection.
+This method is defined in [CAIP-27](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-27.md).
+
+:::note
+MetaMask doesn't support session IDs.
```

Comment:
do we need this note for getSession as well?

Reply:
The same note is already in the description of `wallet_getSession`.